# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pciutils/pciutils-3.1.8-fix-libpci-abi-version-symlink.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Fix to create proper symlinks to shared library

Without this patch the symlink libpci.so -> libpci.so.3 is broken, thus we
create libpci.so.3 -> libpci.so.3.1.8
 
--- pciutils-3.1.8/Makefile.orig	2012-01-13 23:54:01.588075533 +0100
+++ pciutils-3.1.8/Makefile	2012-01-14 00:40:20.924075534 +0100
@@ -110,6 +110,7 @@
 	$(DIRINSTALL) -m 755 $(DESTDIR)$(LIBDIR)
 	$(INSTALL) -c -m 644 lib/$(PCILIB) $(DESTDIR)$(LIBDIR)
 	ln -sf $(PCILIB) $(DESTDIR)$(LIBDIR)/$(LIBNAME).so$(ABI_VERSION)
+	ln -sf $(PCILIB).$(VERSION) $(DESTDIR)$(LIBDIR)/$(LIBNAME).so$(ABI_VERSION)
 endif
 
 install-lib: $(PCIINC_INS) lib/$(PCILIB) lib/$(PCILIBPC)
@@ -119,6 +120,7 @@
 	$(INSTALL) -c -m 644 lib/$(PCILIBPC) $(DESTDIR)$(PKGCFDIR)
 ifeq ($(SHARED),yes)
 	ln -sf $(LIBNAME).so$(ABI_VERSION) $(DESTDIR)$(LIBDIR)/$(LIBNAME).so
+	ln -sf $(LIBNAME).so.$(VERSION) $(DESTDIR)$(LIBDIR)/$(LIBNAME).so$(ABI_VERSION)
 endif
 
 uninstall: all
